dev-docs/bidders/between.md

```diff
@@ -15,6 +15,9 @@ tcf2_supported: true
 {: .table .table-bordered .table-striped }
 | Name          | Scope    | Description | Example | Type     |
 |---------------|----------|-------------|---------|----------|
-| `host` | required |  between host url prefix         | `eu`   | `string` |
+| `host` | required |  Between SSP host url prefix. Defines data center where requests will be sent. Choose the closest one to the prebid-server you are using. Allowed values: `lbs-eu1.ads`, `lbs-ru1.ads`, `lbs-asia1.ads`, `lbs-us-east1.ads` | `lbs-eu1.ads` | `string` |
+| `publisher_id` | required |  Publisher ID from Between SSP control panel | `123` | `string` |
+| `bid_floor` | optional | Minimum price acceptable for a bid. `0.00001` by default  | `0.05` | `float` |
+| `bid_floor_cur` | optional |  Bid floor currency | `EUR` `RUB` `USD` | `string` |
```

Comment:
Sorry I missed this but the same applies for `bid_floor_cur` as well. Can you please mention them as allowed values in the description too and change the example to be only one of the values?

Reply:
Yes, I missed it too :) done